src/helpers.js

```diff
@@ -23,4 +27,23 @@ function createBookings(rawBookingsData, rawCustomersData, rawRoomsData) {
   return bookings;
 }
 
-export default createBookings;
\ No newline at end of file
+function createRooms(rawRoomsData) {
+  const rooms = [];
+
+  const roomsData = rawRoomsData.rooms;
+
+  roomsData.forEach(roomData => {
+    const room = new Room(roomData);
+    rooms.push(room);
+  });
+
+  return rooms;
+}
+
+function formatDate(date) {
+  const prettyDate = date.replace(/-/g, '/');
+  return prettyDate;
+}
```

Comment:
ahh the cursed replace/replaceAll function

Reply:
RIP `replaceAll`